install/rhel-installation-guide.md

```diff
@@ -0,0 +1,32 @@
+# Installing Kata Containers on RHEL
+
+> **Notes:**
+>
+> - Kata Containers packages are available for [RHEL\*](https://www.redhat.com)
+>   version 7 (currently `x86_64` only).
+>
+> - If you are installing on a system that already has Clear Containers or `runv` installed,
+>   first read [the upgrading document](../Upgrading.md).
+>
+> - If you do not want to copy or type all these instructions by hand, you can use the
+>   [`kata-manager`](https://github.com/kata-containers/tests/blob/master/cmd/kata-manager/kata-manager.sh)
+>   script to install the packaged system including your chosen container
+>   manager. Alternatively, you can generate a runnable shell script from
+>   individual documents using the
+>   [`kata-doc-to-script`](https://github.com/kata-containers/tests/blob/master/.ci/kata-doc-to-script.sh) script.
+
+1. Install the Kata Containers components with the following commands:
+
+   > **Note:** This installation channel is not secure since the repository currently
+   > redirects download URLs to `http`.
+
+   ```bash
+   $ source /etc/os-release
+   $ sudo -E VERSION_ID=$VERSION_ID yum-config-manager --add-repo "http://download.opensuse.org/repositories/home:/katacontainers:/release/RHEL_${VERSION_ID}/home:katacontainers:release.repo"
+   $ sudo -E yum -y install kata-runtime kata-proxy kata-shim
+   ```
+
+2. Decide which container manager you will use and select the corresponding link that follows:
```

Comment:
2. Decide which container manager to use and select the corresponding link that follows:

Reply:
Same as above. If we change this here, we need to change it everywhere.

Reply:
@jodh-intel and @klynnrif , I can do the changes in all the other documents. I will do this change in this pr @jodh-intel

Reply:
Yes, thank you @GabyCT!